index.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+const bitmapFileHelper = require(`${__dirname}/lib/bitmap-file-helper.js`);
+const bitmapConstructor = require(`${__dirname}/model/bitmap-constructor.js`);
+const greyScaleTransform = require(`${__dirname}/lib/grey-scale-transform.js`);
+const reverseTransform = require(`${__dirname}/lib/reverse-transform.js`);
+const invertTransform = require(`${__dirname}/lib/invert-transform.js`);
+const infraredTransform = require(`${__dirname}/lib/infrared-transform.js`);
+
+
+bitmapFileHelper(`${__dirname}/assets/palette-bitmap.bmp`, function(err, data) {
```

Comment:
These function calls could be handled modularly in fileHelper. That would allow you to test your whole writing file path easier.